From 161f095bdb597d538c99f062241923ce035b3a8d Mon Sep 17 00:00:00 2001
From: Stanislaw Kardach <skardach@marvell.com>
Date: Tue, 11 Jun 2019 16:49:14 +0200
Subject: [PATCH 072/138] octeontx2-af: enable PTP driver on LOKI platform

commit baa234d5af006ca29967704c706005d59d837521 from
git@git.assembla.com:cavium/WindRiver.linux.git

LOKI platform has its own subsystem device id, therefore add it so that
PTP driver can bind.

Change-Id: I2c95bcabebf0d0bc78811d80a92674a276462cf8
Signed-off-by: Stanislaw Kardach <skardach@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/10734
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/ptp.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/ptp.c b/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
index e96efe95d64d..f340974df163 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
@@ -19,6 +19,7 @@
 #define PCI_DEVID_OCTEONTX2_PTP		0xA00C
 #define PCI_SUBSYS_DEVID_OCTX2_96XX_PTP	0xB200
 #define PCI_SUBSYS_DEVID_OCTX2_95XX_PTP	0xB300
+#define PCI_SUBSYS_DEVID_OCTX2_LOKI_PTP	0xB400
 #define PCI_DEVID_OCTEONTX2_RST		0xA085
 
 #define PCI_PTP_BAR_NO	0
@@ -208,6 +209,9 @@ static const struct pci_device_id ptp_id_table[] = {
 	{ PCI_DEVICE_SUB(PCI_VENDOR_ID_CAVIUM, PCI_DEVID_OCTEONTX2_PTP,
 			 PCI_VENDOR_ID_CAVIUM,
 			 PCI_SUBSYS_DEVID_OCTX2_95XX_PTP) },
+	{ PCI_DEVICE_SUB(PCI_VENDOR_ID_CAVIUM, PCI_DEVID_OCTEONTX2_PTP,
+			 PCI_VENDOR_ID_CAVIUM,
+			 PCI_SUBSYS_DEVID_OCTX2_LOKI_PTP) },
 	{ 0, }
 };
 
-- 
2.17.1

